### DISCUSSION TITLE
Can I hide the page with frontmatter?

### DISCUSSION THREAD

**Discussion by jwonyLee** [shuding/nextra#1103] [Q&A]
Hi, I am using the Nextra Docs theme and I want to know if it's possible to hide a page using frontmatter. Specifically, I want to prevent the page from rendering and only allow access to it through the GitHub repository. like this:

```markdown
---
published: false
---

# Getting Started

Hello, World!
```

Thank you for creating such a great theme.

**Comment by SincerelyFaust** [comment]
You can do this: https://nextra.site/docs/docs-theme/page-configuration#hidden-routes

**Reply by jwonyLee** [reply]
Thank you for the response. However, the problem is that this method can be accessed via url.

**Reply by amedott** [reply]
> You can do this: https://nextra.site/docs/docs-theme/page-configuration#hidden-routes

would this be also possible with the Blog theme, hiding the "posts" page?

**Reply by shuding** [reply]
> the problem is that this method can be accessed via url.

This is not avoidable. In Next.js every valid page is a route by default. You can probably use rewrites or middleware to change the behavior of that route (e.g. require a specific cookie). If you want to make it a draft, I'd suggest naming the file as `my-post.mdx.draft`.
